recipes/gz-sim/meta.yaml

```
@@ -0,0 +1,147 @@
+        - tinyxml2
+        - xorg-libxfixes  # [linux]
+
+    test:
+      commands:
+        - test -f ${PREFIX}/include/gz/{{ component_version }}/gz/{{ component_name }}.hh  # [not win]
+        - test -f ${PREFIX}/lib/lib{{ name }}.so  # [linux]
+        - test -f ${PREFIX}/lib/lib{{ name }}.dylib  # [osx]
+        - test -f ${PREFIX}/lib/cmake/{{ name }}/{{ name }}-config.cmake  # [not win]
+        - if not exist %PREFIX%\\Library\\include\\gz\\{{ component_version }}\\gz\\{{ component_name }}.hh exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\lib\\{{ name }}.lib exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\bin\\{{ name }}.dll exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\lib\\cmake\\{{ name }}\\{{ name }}-config.cmake exit 1  # [win]
+
+  - name: {{ python_name }}
+    script: build_py.sh  # [unix]
+    script: bld_py.bat  # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - swig
+        - ninja
+        - cmake
+        - pkg-config
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - pybind11                               # [build_platform != target_platform]
+        - pybind11-abi                           # [build_platform != target_platform]
+        - pytest                                 # [build_platform != target_platform]
+        - {{ cdt('mesa-libgl-devel') }}  # [linux]
+        - {{ cdt('mesa-dri-drivers') }}  # [linux]
+        - {{ cdt('libselinux') }}  # [linux]
+        - {{ cdt('libxdamage') }}  # [linux]
+        - {{ cdt('libxxf86vm') }}  # [linux]
+        - {{ cdt('libxext') }}     # [linux]
+      host:
+        - {{ pin_subpackage(cxx_name, exact=True) }}
+        - libgz-common5
+        - python
+        - pip
+        - pybind11
+        - pybind11-abi
+        - pytest
+        # Workaround for "package libgz-sim7-7.0.0-hc835698_0 requires qt-main >=5.15.6,<5.16.0a0, but none of the providers can be installed" error
+        # Please remove after migration to qt=5.15
+        - qt-main=5.15
+        - xorg-libxfixes  # [linux]
+      run:
+        - {{ pin_subpackage(cxx_name, exact=True) }}
+        - python
+    test:
+      commands:
+        - pip check
+      requires:
+        - pip
+      imports:
+        - gz.{{ component_name }}{{ major_version }}
+
+  - name: {{ name }}
+    build:
+      run_exports:
+        - {{ pin_subpackage(cxx_name, max_pin='x') }}
+    requirements:
+      run:
+        - {{ pin_subpackage(cxx_name, exact=True) }}
+        - {{ pin_subpackage(python_name, max_pin='x.x.x') }}
+    test:
+      imports:
+        - gz.{{ component_name }}{{ major_version }}
+
+about:
+  home: https://github.com/gazebosim/gz-sim
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: "Open source robotics simulator. The latest version of Gazebo."
```

```suggestion
  summary: "Open source robotics simulator. The 7 version of Gazebo Sim."
```
minor thing expecting new version in the future.

---

> Probably we can remove that sentence altogether?

+1